packages/amqp/README.md

```diff
@@ -0,0 +1,67 @@
+# AMQP (Advanced Message Queuing Protocol)
+
+The library provides support for both direct exchanges and topic exchanges.
+
+> **_NOTE:_**  Check [README.md](../../README.md) for transport-agnostic library documentation.
+>
+
+## Publishers
+
+Use `AbstractAmqpQueuePublisher` to implement direct exchange and `AbstractAmqpTopicPublisher` for topic exchange.
```

Comment:
we also need to document out PublisherManager functionality, but that's a separate big topic

Reply:
I've created a separate ticket to not stall this one -> https://lokalise.atlassian.net/browse/MET-53